Swiftfin/Views/SettingsView/UserDashboardView/DevicesView/DevicesView.swift

```diff
@@ -0,0 +1,245 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import JellyfinAPI
+import SwiftUI
+
+// TODO: Replace with CustomName when Available
+
+struct DevicesView: View {
+
+    @EnvironmentObject
+    private var router: SettingsCoordinator.Router
+
+    @StateObject
+    private var viewModel: DevicesViewModel
+
+    @State
+    private var isPresentingDeleteSelectionConfirmation = false
+    @State
+    private var isPresentingDeleteConfirmation = false
+    @State
+    private var isPresentingSelfDeleteError = false
+    @State
+    private var selectedDevices: Set<String> = []
+    @State
+    private var editMode: Bool = false
```

Comment:
```suggestion
    private var isEditing: Bool = false
```

Although, I should refactor to use the environment `editMode`.